docs/resources/Channel.md

```diff
@@ -264,7 +264,7 @@ Represents a message sent in a channel within Discord.
 > Fields specific to the `MESSAGE_CREATE` and `MESSAGE_UPDATE` events are listed in the [Gateway documentation](#DOCS_TOPICS_GATEWAY_EVENTS/message-create).
 
 > warn
-> `content`, `embeds`, `attachments`, and `components` require the [`MESSAGE_CONTENT` intent](#DOCS_TOPICS_GATEWAY/message-content-intent) to receive non-empty values.
+> `content`, `embeds`, `attachments`, `components`, and `poll` require the [`MESSAGE_CONTENT` intent](#DOCS_TOPICS_GATEWAY/message-content-intent) to receive non-empty values.
```

Comment:
Should polls also be added in the list in [Gateway docs](https://github.com/discord/discord-api-docs/blob/cf50243f0a4c87a62c124811c8a7f28d97176a61/docs/topics/Gateway.md?plain=1#L449) or is that list not considered complete, because it says "For example"